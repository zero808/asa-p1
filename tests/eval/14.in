10000 985
10 287
22 1683
26 2817
29 5482
31 3640
37 9420
47 8875
57 5313
86 7244
123 9931
127 3421
133 783
137 5869
144 5637
151 5079
159 5545
160 1426
186 5734
187 4353
206 8421
207 5742
214 2607
236 4652
246 9307
250 6460
257 7686
259 6259
288 8055
297 5688
310 4725
319 5009
329 2587
337 8123
342 9547
348 6933
350 8536
368 3134
387 8761
395 6794
408 4313
409 43
416 5168
417 7218
428 4888
436 6397
448 2478
474 7703
484 323
503 2424
530 5614
550 8962
564 1991
564 3654
569 1145
607 4050
624 6183
632 5174
632 8679
647 7613
654 5890
671 1897
715 1334
725 1064
737 7255
739 5362
745 55
752 4426
755 2659
769 2200
775 5079
777 3039
785 3100
796 1940
815 5423
824 7332
827 2205
832 1133
843 612
847 5796
850 4429
855 7466
865 1821
872 8968
880 6035
882 2725
908 6288
927 8736
930 3418
934 1815
937 8844
966 9833
976 3043
984 5781
1015 9984
1024 9297
1029 2164
1036 1043
1042 6194
1064 219
1079 4523
1085 7990
1095 6796
1104 4611
1104 7633
1106 9584
1129 4790
1136 2310
1149 2297
1153 136
1153 1127
1154 4670
1165 7447
1174 1349
1181 6970
1193 7803
1195 1485
1209 3296
1237 5647
1243 960
1256 1331
1259 8590
1266 9297
1284 9835
1291 8765
1293 9113
1309 624
1311 8871
1319 5744
1346 6774
1357 1726
1357 8336
1359 1941
1367 3721
1369 4059
1371 2997
1389 4228
1403 2185
1408 2978
1423 3059
1431 4142
1475 3680
1480 2100
1481 7406
1488 5440
1489 8276
1491 326
1495 6980
1497 9737
1502 5661
1506 8882
1522 69
1528 6007
1535 1198
1552 8262
1569 1304
1594 5984
1614 4444
1615 5665
1620 3962
1621 7255
1634 9073
1648 9991
1665 144
1701 2663
1705 1023
1705 1500
1724 6694
1732 7512
1732 8324
1734 7
1750 3802
1751 6217
1752 4843
1772 7046
1783 1692
1790 4682
1796 4824
1800 6016
1812 794
1822 3424
1835 2840
1838 4273
1847 8363
1861 2709
1867 5870
1873 4235
1875 7054
1882 5587
1884 8003
1885 9481
1895 670
1899 2443
1912 552
1925 395
1945 8936
1961 2855
1962 8953
1971 1419
1990 7196
2026 6431
2028 1042
2079 5705
2080 9042
2107 4182
2112 4027
2113 6779
2129 6630
2151 9344
2151 9395
2160 5748
2165 160
2180 538
2210 1906
2218 5665
2227 3810
2227 9759
2245 1804
2263 3319
2270 9491
2272 4178
2274 2844
2281 1491
2286 3143
2298 1265
2302 4862
2308 5332
2312 5641
2320 1808
2323 4542
2323 5603
2339 5594
2348 4644
2363 7047
2384 7981
2405 2773
2407 9764
2410 4299
2414 7450
2418 557
2420 7577
2442 2767
2447 7196
2454 6581
2456 4882
2462 4870
2473 6405
2480 9705
2494 3704
2516 7277
2519 1482
2548 9542
2550 9412
2550 9705
2564 6720
2567 4780
2575 5698
2584 2552
2591 8459
2596 1036
2596 1257
2598 2955
2626 658
2661 1534
2668 4964
2669 7698
2682 1661
2693 4608
2706 7042
2713 7008
2734 6535
2747 9978
2751 3344
2752 8916
2756 3857
2764 6264
2767 4814
2771 7422
2774 4594
2774 8225
2780 8275
2796 9811
2801 1814
2806 1346
2826 4863
2831 512
2847 2936
2860 7894
2862 524
2863 2066
2867 8293
2871 121
2892 4060
2896 2230
2898 1991
2903 322
2910 4726
2934 5571
2950 8686
2971 595
3002 7261
3021 3182
3027 5919
3029 8574
3039 1826
3047 8693
3050 9482
3052 2341
3068 4690
3074 1659
3084 8935
3095 7838
3099 3225
3104 9776
3112 8055
3146 2748
3153 9578
3158 1381
3161 6413
3167 7012
3241 5344
3244 4349
3248 5491
3267 5386
3290 5672
3296 4552
3298 9734
3306 1273
3313 7968
3314 5260
3317 574
3318 8457
3332 5470
3338 5557
3343 5270
3349 7602
3349 9192
3350 2120
3353 3345
3357 7569
3367 934
3368 6775
3377 3896
3380 58
3387 9990
3391 9207
3405 505
3412 9099
3428 2459
3437 5374
3449 6443
3456 3553
3473 2788
3484 6569
3496 3153
3501 1618
3522 7691
3528 4976
3630 1696
3632 4671
3639 1525
3645 617
3646 2149
3652 747
3652 7129
3678 5387
3691 7705
3712 2368
3716 8363
3740 2926
3741 3638
3745 9197
3754 9255
3768 3630
3771 8003
3772 3215
3774 5789
3785 2472
3797 6551
3799 4797
3802 4987
3810 6054
3814 6330
3817 8517
3836 545
3875 9757
3877 6852
3882 9814
3884 7240
3900 6265
3911 3973
3925 6402
3938 9305
3939 7000
3961 8790
3962 6739
3965 7553
3974 4579
3982 3762
3987 3562
4000 9090
4005 2311
4041 4112
4041 6229
4043 7974
4044 3720
4060 5253
4062 4522
4070 7420
4106 9610
4107 5341
4108 8407
4122 9071
4127 6546
4144 290
4144 3377
4159 591
4160 723
4161 2259
4170 5378
4171 2562
4177 6808
4191 2498
4210 2429
4216 9612
4225 2047
4242 2347
4257 8338
4267 9546
4297 7335
4298 3092
4323 7033
4325 4221
4328 3642
4331 5022
4334 5943
4352 49
4361 1043
4381 4702
4400 153
4406 3194
4409 4510
4417 8536
4418 6468
4428 9322
4436 5516
4453 9890
4471 3609
4473 4088
4476 4394
4481 2028
4483 4275
4494 7907
4495 9870
4530 1282
4533 3663
4543 3711
4546 9619
4549 7246
4550 757
4562 2592
4563 2044
4583 570
4591 2613
4613 5696
4630 8972
4633 895
4639 5804
4642 1720
4650 898
4662 6775
4665 6102
4665 9892
4670 4734
4689 9933
4709 771
4721 8153
4722 2350
4736 3784
4763 5515
4764 8712
4767 3858
4768 9422
4787 933
4788 3371
4789 6120
4792 237
4804 8430
4806 3990
4826 6356
4828 9538
4857 763
4858 280
4869 3455
4882 4712
4889 6769
4953 4920
4973 9040
4989 660
4992 7384
5003 2049
5006 7437
5024 6026
5031 6333
5042 5678
5046 7408
5049 1502
5054 336
5059 9607
5068 2943
5069 5538
5076 7800
5077 2580
5078 716
5080 1912
5082 4359
5089 1674
5094 3517
5097 7163
5100 4670
5108 5274
5119 1796
5126 8038
5133 113
5153 2480
5153 7902
5154 7039
5164 8944
5173 1215
5188 2399
5207 4450
5207 5579
5235 1580
5256 4247
5274 2507
5288 1153
5288 1194
5294 6395
5299 1839
5311 4997
5328 6109
5334 4197
5341 4750
5342 5507
5354 1972
5364 9669
5371 8151
5376 1487
5393 5825
5398 1515
5405 7015
5407 7135
5408 9527
5412 528
5420 9869
5425 9191
5432 671
5454 2332
5457 7790
5465 6031
5480 5372
5481 1043
5502 1327
5518 9544
5524 693
5532 1287
5548 7932
5557 8487
5571 2595
5590 5735
5609 4855
5612 9463
5619 6620
5637 830
5667 1056
5669 6913
5682 8386
5686 2697
5698 8996
5717 3408
5727 1580
5731 8170
5746 2596
5778 8768
5796 2801
5796 3635
5801 7731
5807 5726
5840 5919
5848 9226
5848 9911
5850 9634
5857 706
5887 3415
5887 4157
5911 5576
5927 4390
5938 7873
5952 578
5960 6756
5963 2954
5975 3564
5975 4663
5996 6702
6000 6745
6002 2850
6015 2623
6016 5341
6027 106
6027 1749
6029 5806
6043 2314
6050 3589
6055 925
6060 6702
6064 3833
6093 1686
6103 9334
6111 3993
6116 7468
6125 8024
6129 899
6138 9398
6140 7301
6144 1828
6164 9360
6169 2419
6183 7666
6188 2510
6191 6357
6192 2217
6204 7686
6205 5348
6207 7173
6213 7124
6217 2777
6231 2732
6235 4508
6246 6405
6284 4431
6311 8344
6329 7713
6333 385
6340 410
6340 2902
6366 9156
6370 8223
6371 7385
6375 6192
6375 9170
6383 2425
6388 2920
6394 525
6425 7082
6433 9015
6435 9589
6438 677
6447 3134
6447 6155
6450 1123
6498 8291
6500 6449
6507 6643
6518 5673
6533 4663
6540 5090
6542 9618
6547 1955
6551 2377
6553 8379
6579 9238
6642 1952
6652 7150
6655 5521
6669 9511
6670 7648
6672 8619
6721 7796
6741 6562
6774 3971
6794 9037
6810 8660
6823 3854
6825 7021
6826 9610
6847 665
6868 2156
6870 4270
6886 3574
6896 3295
6900 9075
6901 9399
6907 5847
6922 6880
6930 3449
6930 8191
6931 1141
6933 440
6935 9107
6937 6690
6948 5001
6962 1832
6983 1651
6991 9172
7004 1513
7008 2501
7020 7306
7039 1003
7058 3988
7064 5562
7074 8333
7087 7907
7105 3776
7109 1066
7109 9042
7111 452
7120 1089
7121 6013
7150 4234
7156 6922
7159 3440
7164 7971
7164 9064
7179 9200
7187 2960
7212 2007
7230 1493
7231 3030
7235 1625
7235 5341
7260 8508
7284 3448
7302 8543
7321 2253
7326 7543
7349 3686
7360 297
7365 6121
7373 6800
7374 3550
7398 1720
7400 8506
7407 2340
7407 4463
7430 8815
7454 6571
7457 5529
7457 8816
7458 9002
7491 5126
7492 9310
7500 7264
7502 574
7519 6552
7527 3109
7528 6187
7535 543
7563 4765
7563 9293
7594 5553
7602 5452
7610 2921
7623 6358
7627 5176
7649 7538
7654 756
7657 2016
7662 3568
7666 7031
7682 3929
7685 8885
7690 1387
7693 8018
7703 7110
7706 8037
7722 5793
7742 3114
7746 7732
7749 3745
7751 8504
7753 7466
7757 6627
7757 7710
7786 9410
7790 5934
7794 2628
7820 5848
7825 1398
7873 9792
7880 9218
7887 8017
7908 2323
7916 9810
7923 9110
7931 3496
7937 1106
7958 7057
7962 831
7969 2695
7969 4067
7976 9388
7981 7179
8002 3043
8003 3592
8028 2711
8050 9393
8056 8312
8095 9020
8097 5936
8106 267
8110 8696
8179 2808
8192 2752
8211 9204
8219 1307
8242 7378
8268 4027
8271 2951
8272 7660
8283 7047
8295 6535
8326 9624
8341 6651
8369 4233
8391 3243
8411 10000
8416 7511
8418 7328
8444 7217
8445 2677
8445 3563
8467 7020
8469 4997
8480 6435
8485 8585
8501 9438
8518 4921
8541 1989
8541 8243
8553 5295
8556 4845
8560 1568
8564 7597
8571 4978
8571 8368
8584 3206
8600 9804
8652 5985
8655 5618
8708 9589
8722 5386
8740 4209
8745 8423
8761 740
8762 1582
8790 7397
8802 8776
8821 2135
8821 2202
8822 5498
8826 2205
8829 508
8855 8944
8862 9622
8863 9019
8867 4161
8881 8526
8889 1083
8900 8875
8902 8364
8905 6466
8942 4126
8954 8574
8958 7646
8967 6513
8988 1437
8995 1938
8996 621
9007 3804
9020 5702
9023 2042
9033 7822
9035 2927
9035 8226
9046 6249
9053 7630
9066 7303
9066 7967
9073 7444
9077 956
9083 4796
9089 3999
9092 1683
9098 1646
9122 1997
9122 3105
9129 1240
9132 3620
9141 5309
9143 3930
9152 9167
9155 9516
9172 6385
9184 6787
9194 5617
9200 1607
9200 4740
9201 9941
9205 9233
9215 8917
9234 2867
9238 7878
9247 6341
9254 9251
9266 9835
9280 8362
9285 9181
9293 8737
9295 4161
9297 671
9315 81
9326 3497
9328 741
9335 2976
9341 1397
9360 4298
9410 3269
9418 6694
9419 677
9434 176
9435 7244
9440 7802
9464 5247
9470 9605
9472 4461
9500 9287
9502 5642
9503 9659
9513 188
9514 3971
9523 8357
9540 4187
9553 9297
9599 7912
9620 6336
9637 7370
9641 6647
9653 8124
9659 2418
9664 1168
9671 96
9674 8703
9682 1925
9695 3373
9704 6791
9711 3552
9711 6854
9762 4470
9774 4814
9789 4746
9793 1917
9798 2082
9800 3407
9806 9304
9810 5714
9823 312
9823 1392
9830 2933
9833 7948
9836 2563
9848 7423
9853 6224
9858 26
9863 9528
9870 7147
9872 2185
9876 4614
9876 6291
9888 5606
9889 6200
9892 6142
9932 2273
9978 6943
9981 5316
9998 9853
1998 2001
