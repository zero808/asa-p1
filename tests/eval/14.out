10000
1
9064
